cli/src/lib/initialize.ts

```diff
@@ -62,22 +62,6 @@ export default async function Initialize(
 
   safelyCreateFile("gitignore", ".gitignore"); // we need to call the source file something other than `.gitignore` so it's not ignored by NPM
 
-  /**
```

Comment:
Since Config UI generates the directories it needs, we don't need this skeleton structure anymore.  The `init` command directs folks to the Config docs, so the READMEs that were housed in the empty folders aren't needed either.